### PR TITLE
d8_to_d8: D8 to D8 migration sample module is added.

### PR DESCRIPTION
**Pull request by pranitjha**

### CODE REVIEW

File: modules/examples/d8_to_d8_migrate/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Drupal 8 to Drupal 8 Migrate
+
+This is a sample module to migrate data from Drupal 8 database to drupal 8.
```
Comment (joshirohit100) [inline modules/examples/d8_to_d8_migrate/README.md]: **database** word is not required.

**Comment by joshirohit100** [comment]
@pranitjha what about the term thing? Is that migrated properly? as in 8.5 and 8.6, we have different table for the term heiacrhy.

File: modules/examples/d8_to_d8_migrate/README.md (hunk 1 above)
Comment (pranitjha) [reply modules/examples/d8_to_d8_migrate/README.md]: removed 'database'.

**Comment by pranitjha** [comment]
@joshirohit100 For now, I migrated the terms on the same level without pid. And it is working fine. For adding pid, we need to write a new plugin.

**Comment by joshirohit100** [comment]
But what about the case we discussed yesterday where source was 8.6 ans destination was 8.5, we had to change the process plugin?

Also will it work for the case of 8.5 => 8.6?

**Comment by pranitjha** [comment]
For both the case, we need to write the plugin as we are having a different structure in both versions. So, we can keep this as TODO for future, as it will take significant time now.

**Comment by joshirohit100** [comment]
so pls do this for the 8.6 and override the source plugin

**Comment by pranitjha** [comment]
Closing this as I've raised another PR with all the modifications.

**Pull request closed without merging**